packages/ruff-lsp/package.yaml

```
@@ -9,6 +9,10 @@ languages:
 categories:
   - LSP
 
+deprecation:
+  since: v0.0.61
+  message: `ruff server` supports the same feature set as ruff-lsp, but with superior performance and no installation required. ruff server was marked as stable in Ruff v0.5.3.
```

Just to be sure, it could make sense to explicitly mention that the package is also available at mason with the name `ruff`?

---

```suggestion
  message: please use `ruff` as a replacement. `ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no installation required. ruff server was marked as stable in Ruff v0.5.3.
```

Could this be acceptable wording?

---

How about 

> please use the mason package `ruff` as replacement. The command `ruff server`